### PR TITLE
[Draft, don't merge] Fix failing windows

### PR DESCRIPTION
**Pull request by LysandreJik**
Tests that the fix on `transformers` fixes the CI here

### CODE REVIEW

File: .github/workflows/tests-main.yml [hunk 1]
```diff
@@ -30,7 +30,7 @@ jobs:
         python -m pip install --upgrade pip
         # install PEFT & transformers from source
         pip install -U git+https://github.com/huggingface/peft.git
-        pip install -U git+https://github.com/huggingface/transformers.git@7f112caac2ff365c3d6e0020fefe8c1300311e07 # temporary fix for transformers, remove once the import issue on windows is solved 
+        pip install -U git+https://github.com/huggingface/transformers.git@25d1d3862086f76741b150682d27c19e8b69c828
```
Comment (qgallouedec) [inline .github/workflows/tests-main.yml:33]:
```suggestion
        pip install -U git+https://github.com/huggingface/transformers.git
```

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -58,7 +58,7 @@ jobs:
         python -m pip install --upgrade pip
         # install PEFT & transformers from source
         pip install -U git+https://github.com/huggingface/peft.git
-        pip install -U git+https://github.com/huggingface/transformers.git@7f112caac2ff365c3d6e0020fefe8c1300311e07 # temporary fix for transformers, remove once the import issue on windows is solved 
+        pip install -U git+https://github.com/huggingface/transformers.git@25d1d3862086f76741b150682d27c19e8b69c828
```
Comment (qgallouedec) [inline .github/workflows/tests.yml:61]:
```suggestion
        pip install -U git+https://github.com/huggingface/transformers.git
```

**Comment by LysandreJik** [comment]
cc @qgallouedec it seems like that fixed it!

**Comment by qgallouedec** [comment]
Thanks a lot @LysandreJik

**Pull request merged**